corehq/apps/users/tasks.py

```diff
@@ -295,10 +295,20 @@ def reset_demo_user_restore_task(commcare_user_id, domain):
 
 @task(serializer='pickle')
 def remove_unused_custom_fields_from_users_task(domain):
-    from corehq.apps.users.custom_data import (
-        remove_unused_custom_fields_from_users,
+    """Removes all unused custom data fields from all users in the domain"""
+    from corehq.apps.custom_data_fields.models import CustomDataFieldsDefinition
+    from corehq.apps.users.dbaccessors import get_all_commcare_users_by_domain
+    from corehq.apps.users.views.mobile.custom_data_fields import (
+        CUSTOM_USER_DATA_FIELD_TYPE,
     )
-    remove_unused_custom_fields_from_users(domain)
+    fields_definition = CustomDataFieldsDefinition.get(domain, CUSTOM_USER_DATA_FIELD_TYPE)
+    assert fields_definition, 'remove_unused_custom_fields_from_users called without a valid definition'
```

Comment:
is this message still correct? Does the PR not remove `remove_unused_custom_fields_from_users`. Might be misleading when debugging.